persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/ConversationDAOImpl.kt

```diff
@@ -569,9 +574,14 @@ class ConversationDAOImpl(
             conversationQueries.updateConversationType(type, conversationID)
         }
 
-    override suspend fun updateConversationProtocol(conversationId: QualifiedIDEntity, protocol: ConversationEntity.Protocol) {
-        withContext(coroutineContext) {
-            conversationQueries.updateConversationProtocol(protocol, conversationId)
+    override suspend fun updateConversationProtocol(conversationId: QualifiedIDEntity, protocol: ConversationEntity.Protocol): Boolean {
```

Comment:
we can simplify this via having the update query and the get query on sql leve.

Reply:
Cool, can you show me how?

Reply:
Figured it out.